build-parent/pom.xml

```diff
@@ -840,12 +840,16 @@
 
                         <configuration>
                             <kotlin>
+                                <ktfmt>
+                                    <style>KOTLINLANG</style>
```

Comment:
Dumb question: do the other styles generate code that are more like we used to have?

Reply:
i played with the others and they're all ending up touching quite a few files even it's only changing indentation.  But mostly it's variance in wrapping rules and stacked annotations vs all one on line kinds of things.